src/components/FormView.jsx

```diff
@@ -148,6 +150,18 @@ const FormView = ({
                   </Grid.Row>
                 )}
 
+                {process.env.RAZZLE_HCAPTCHA_KEY && (
+                  <Grid.Row>
+                    <Grid.Column>
+                      <HCaptcha
+                        sitekey={process.env.RAZZLE_HCAPTCHA_KEY}
+                        onVerify={onVerifyCaptcha}
+                        size="invisible"
```

Comment:
This should probably be an option for whoever uses this addon. The invisible captcha has privacy policy implications: [see hcaptcha suggestions here](https://docs.hcaptcha.com/faq#do-i-need-to-display-anything-on-the-page-when-using-hcaptcha-in-invisible-mode)